### ISSUE TITLE
Add and support WOFF2 fonts

### ISSUE THREAD

**Issue by TomasRiker** [mathjax/MathJax#1990]
The WOFF2 format for web fonts "offers a 30% average gain over WOFF 1.0 (up to 50%+ in some cases)" according to the developers. It would be nice if WOFF2 was supported by MathJax.

**Comment by dpvc** [comment]
We will be redoing the fonts for MathJax v3 and will consider WOFF2 for that.  Thanks for the suggestion.

**Comment by sanmai-NL** [comment]
@dpvc `mathjax-full/es5/output/chtml/fonts/woff-v2` does not contain WOFF2 but WOFF fonts?

**Comment by sanmai-NL** [comment]
I’m getting decoding errors because the extensions are `woff`, not `woff2` so the webservers sets the content-type to `font/woff` instead of `font/woff2`.

**Comment by dpvc** [comment]
@sanmai-NL, yes, the fonts are WOFF not WOFF2.  The name `woff-v2` refers to these being the MathJax v2 fonts (in WOFF format), not to version 2 of WOFF.  This was in order to get MathJax v3 up and running with the fonts that we had, but with plans to redo the MathJax fonts during the font update that we are currently working on.  That is a time-consuming process, and we hope to have a release with new fonts at the end of the summer.

> the webserver sets the content type to `font/woff` instead of `font/woff2`.

That is correct, as they really *are* WOFF not WOFF2.  I am not able to reproduce the errors you reported in #2714.

**Comment by dpvc** [comment]
Fixed in v4.0 (the fonts are WOFF2 format).
